### PR TITLE
Grammar fix for new network pages

### PR DESCRIPTION
**Pull request by fhildeb**

### CODE REVIEW

File: docs/networks/mainnet.md [hunk 1]
```diff
@@ -34,30 +35,37 @@ LUKSO will start as a PoS Blockchain with an Initial Genesis Validator Set that
...
 
-The second file, `genesis.json`, contains the initial balances of the network. This will include the initial balances and total supply of LYX. The community will have the choice of setting the initial supply at 35M, 42M (the LUKSO team's suggestion), or 100M.
+The second file, `genesis.json` contains the initial balances of the network. It will include the initial balances and total supply of LYX. The community will choose to set the initial supply at 35M, 42M (the LUKSO team's suggestion), or 100M.
 
-As a Genesis Validator you will need to the run the network clients ahead of time with the downloaded genesis files. The [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli) will make this process easy, and will be provided ahead of time.
+As a Genesis Validator, you will must run the network clients ahead of time with the downloaded genesis files. The [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli) will make this process easy and will be provided ahead of time.
```
Comment (Hugoo) [inline docs/networks/mainnet.md:68]: "you will must" ?

File: docs/networks/mainnet.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ LUKSO's Blockchain Architecture runs the Ethereum protocol and consists of 2 to
 - The consensus client running Casper the Friendly Finality Gadget (Casper FFG) plus LMD-GHOST fork choice algorithm ([More on the Gasper Consensus](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/gasper/))
 - The execution client, [running the Ethereum Virtual Machine](https://ethereum.org/en/developers/docs/ethereum-stack/)
 
-If you are running a validating node you will also run:
+If you are running a validating node, you will also run the following:
```
Comment (Hugoo) [inline docs/networks/mainnet.md:26]: i think the initial sentence is ok

File: docs/networks/l14-testnet.md [hunk 3]
```diff
@@ -34,15 +34,15 @@ The LUKSO L14 test network requires most of [Constantinople transitions](https:/
 
 ### Start the Node
 
-Run your parity node and connect to the LUKSO L14 test network using:
+Run your parity node and connect to the LUKSO L14 test network using the following command:
```
Comment (Hugoo) [inline docs/networks/l14-testnet.md:37]: not sure if necessary

**Review by Hugoo — APPROVED** [review]

**Comment by paganys** [comment]
@fhildeb @Hugoo Have you checked my suggestions?

**Pull request merged**